src/common/i18n/locales/en-US.json

```diff
@@ -1794,7 +1794,59 @@
+      "what-friends-feed-body": "<p>Friends feed contains information about posts and reblogs of the accounts you follow.</p>",
+      "how-to-reblog-header": "How to Reblog post on Ecency?",
+      "how-to-reblog-body": "<p>Click on the Rectangular Arrow Icon under the post you like. Then choose Reblog and this post will appear in your Blog.</p>",
+      "what-is-reblog-header": "What is Reblog?",
+      "what-is-reblog-body": "<p>A reblog is when you see a post you enjoy on Ecency, and by clicking the reblog button, you make the post appear on your blog too. Now all of your followers can enjoy the post.</p>",
+      "how-sort-feed-header": "How-to-sort-Friends-feed?",
+      "how-sort-feed-body": "<p>Your Friends Feed contains posts published by your friends and their reblogs. If you only want to see the posts of your friends, you can select Exclude Reblog through the sorting function.</p>",
+      "what-trending-feed-header": "What does the Trending feed show?",
+      "what-trending-feed-body": "<p>Trending feed shows widely mentioned or discussed posts of entire platform.</p>",
+      "how-sort-trending-header": "How to sort Trending feed?",
+      "how-sort-trending-body": "<p>Trending feed can be sorted depending on what information you want to see. For example, you can choose one depending on whether you want to see only posts from My Community or Global posts from the entire platform.</p>",
+      "how-search-tags-header": "How to search content by tags?",
+      "how-search-tags-body": "<p>On the left side of your feed there are mostly used tags shown for fasten your search. By clicking on the tag you can see posts that used this tag.</p>",
+      "what-hot-feed-header": "What does the Hot feed show?",
+      "what-hot-feed-body": "<p> Popular posts in last couple hours are shown in Hot Feed.</p>",
+      "what-information-feed-header": "What kind of informations contain Feed?",
+      "what-information-feed-body": "<p>Feed includes brief information about Post author, Post title, which Community it was published in, post rewards, how many upvotes and comments it get. To perform additional activity with the post, just click on the three dots under the post.</p>",
+      "what-new-feed-header": "What does the New feed show?",
+      "what-new-feed-body": "<p> The newest posts sorted by the published time are shown in the New feed. </p>",
+      "how-change-design-header": "How to change feed design?",
+      "how-change-design-body": "<p>Click on the grid icon at the top right of the feed. Choose one of the look designs: Classic or Grid. </p>",
+      "what-is-deck-header": "What is Deck wiew on Ecency?",
+      "what-is-deck-body": "<p>From other Classic and Grid wiews there is another wiew called Deck. Decks makes easy to manage your account. With decks not only you get to see multiple things in parallel but you can configure each decks and shuffle things around.</p>",
```

Comment:
Typo, view not ~~wiew~~. Also we are separating Decks from View, so in upcoming version there won't be possibility to change view to Deck from dropdown, we are making it separate page like ecency.com/decks.